src/assets/TestFramework/Deployment.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+using System;
+
+namespace Azure.ResourceManager.TestFramework
+{
+    /// <summary> A Class representing a Deployment along with the instance operations that can be performed on it. </summary>
+    public partial class Deployment : ArmResource
+    {
+        /// <summary> Initializes a new instance of the <see cref="Deployment"/> class for mocking. </summary>
+        public Deployment()
+        {
+        }
+
+        public DeploymentData Data;
+    }
+
+    public class DeploymentData {
```

Comment:
This has been renamed to `ArmDeploymentData`
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resources/Azure.ResourceManager.Resources/src/Generated/ArmDeploymentData.cs